.github/workflows/pkgci_regression_test.yml

```diff
@@ -95,6 +97,7 @@ jobs:
         if: "!cancelled()"
         run: |
           source ${VENV_DIR}/bin/activate
+          [ -z "$IREE_TEST_FILES" ] && unset IREE_TEST_FILES
```

Comment:
What are these `unset` commands trying to do? They caused the CPU tests to redownload files and timeout/fail: https://github.com/iree-org/iree/actions/runs/13979151202/job/39140779239?pr=20260#step:6:142

Reply:
If I don't unset IREE_TEST_FILES, the when I try to move the envionmental variable over it sets _REE_TEST_FILES to ''